extension/src/popup/components/sendPayment/SendSettings/SettingsFail/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Button } from "@stellar/design-system";
+import { useTranslation } from "react-i18next";
+
+import { navigateTo } from "popup/helpers/navigate";
+import { ROUTES } from "popup/constants/routes";
+import { View } from "popup/basics/layout/View";
+import IconFail from "popup/assets/icon-fail.svg";
+
+import "./styles.scss";
```

Comment:
this will probably trigger an eslint error - should be the last import

Reply:
Hmm, didn't for me but I'll move it anyways.